app/scripts/controllers/table-search-controller.js

```diff
@@ -211,6 +213,20 @@ angular.module('groongaAdminApp')
         }
       }
 
+      function toggleAllColumns() {
+        $scope.table.allColumns.forEach(function(columnInfo) {
+          columnInfo.output = $scope.toggleAllColumnsChecked;
+        });
+        if (!$scope.toggleAllColumnsChecked) {
+          $scope.table.allColumns.filter(function(columnInfo) {
```

Comment:
I regard that checking the stored value and the key as most cases.
But after a while, I've reconsidered that it is *just one case*, and this behaviour bothers other groonga-admin user's. 
I'll fix it again.